packages/structured-clone/build.js

```diff
@@ -27,6 +27,7 @@ import fs from "node:fs/promises";
     esbuild
         .build({
             ...commonConfig,
+            external: commonConfig.external.filter(dep => !new RegExp(packageJson.internalDependencies).exec(dep)),
```

Comment:
This is a big improvement, but I was hoping for a separate function that was imported. A regex is fine for now, but the long-term solution should be a function that scans the `package.json` of the deps and figures out whether they are `type: "module"` and uses that to decide.

Reply:
I suppose you mean 2ce782f ? Done 👍